views/addInstance.ejs

```diff
@@ -278,6 +278,8 @@
 							$("#newUrl").val(instance.url);
 							$("#newDesc").val(instance.description);
 							$("#maintainerOrgName").val(instance.maintainerOrgName);
+							$("#maintainerEmail").val(instance.maintainerEmail);
+							$("#maintainerGithubUrl").val(instance.maintainerGithubUrl);
 							$("#maintainerUrl").val(instance.maintainerUrl);
```

Comment:
Don't we have to add here as well https://github.com/intermine/intermine-registry/blob/dev/views/addInstance.ejs#L178 ?

Reply:
No, I think because in the issue the only update was mentioned. If you want me to add, I can add it. Should I add it then?

Reply:
I think we should. @danielabutano any thoughts?

Reply:
@arunans23 , yes it is another issue which occurs while a warning is generated whenever any existed instance is added. Fixing it too!

Reply:
Done @arunans23 , you can review the PR now.

Reply:
Good spot @arunans23 ! thanks for the contribution @raghav4